src/intTest/resources/fhir/multi_pathology_msg1.fhir.json

```diff
@@ -1,23 +1,23 @@
 {
     "resourceType": "Bundle",
-    "id": "800ff80a-1bac-4648-840c-1a7f530ac017",
+    "id": "f6afe7cd-090a-4b97-99cb-088e12f821a6",
     "meta": {
-        "lastUpdated": "2021-06-07T10:18:28.432+01:00",
+        "lastUpdated": "2021-06-08T10:47:52.871+01:00",
```

Comment:
I think we may have to create a task ticket to have these kind of timestamps a fixed date. Imagine it's annoying having to update them to the current timestamp each time the integration tests are ran

Reply:
It's not that I have to update it but I copy paste the result replacing whole file. That's why it got updated here